x-pack/test/case_api_integration/spaces_only/tests/common/cases/reporters/get_reporters.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../../../common/ftr_provider_context';
+
+import { getPostCaseRequest } from '../../../../../common/lib/mock';
+import {
+  createCase,
+  deleteCasesByESQuery,
+  getAuthWithSuperUser,
+  getReporters,
+} from '../../../../../common/lib/utils';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+  const authSpace1 = getAuthWithSuperUser();
+
+  describe('get_reporters', () => {
+    afterEach(async () => {
+      await deleteCasesByESQuery(es);
+    });
+
+    it('should not return reporters when security is disabled', async () => {
```

Comment:
This is because if we see `username === null` we don't return it from the backend.